_mappy-breakpoints.scss

```diff
@@ -80,7 +80,7 @@ $mappy-queries: () !default;
   }
   $_return: (
     type: $type, 
-    media-string: inspect($media-string), 
+    media-string: $media-string, 
```

Comment:
That was the bit that caused the error :) I've also tested this new version with Ruby Sass 3.4.21 and works correctly